arcgis-ios-sdk-samples/Edit data/Add features with contingent values/ContingentValuesTableViewController.swift

```diff
@@ -0,0 +1,331 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+protocol ContingentValuesDelegate: AnyObject {
+    func createBufferGraphics()
```

Comment:
- Better to follow the standard naming pattern as something did finish
- It seems this method should not create all the graphics, but rather only the graphic for the newly added feature. Can you split it into 2 method, one for initializing existing graphics when the geodatabase loads, the other for adding 1 graphic when the contingent value process finishes?

Reply:
I think I might have made a wrong suggestion here - no need to change, I'll take another look in next round of review.
It seems like we can have a function to create one buffer graphic for each bird nest feature, which takes the feature and its map point as parameter. So that
- when the geodatabase loads, use a forEach to create graphics for existing features
- when new feature gets added to the geodatabase, call the function in the delegate